firmware/applications/SignBuddy/sensors.c

```diff
@@ -0,0 +1,48 @@
+#include "sensors.h"
+
+#include "ble_uart.h"
+#include "common.h"
+
+#include <stdlib.h>
+
+#define FLEX_SENSOR_CNT      3
+#define PROCESS_PERIOD_MS    1000
+
+typedef struct
+{
+  float sensor_data[FLEX_SENSOR_CNT];
+} flex_data_t;
+
+typedef struct
+{
+  uint32_t last_ticks;
+} state_t;
+
+static state_t state;
+
+int get_flex_data(flex_data_t *data)
+{
+  for (int i = 0; i < FLEX_SENSOR_CNT; i++) {
+    data->sensor_data[i] = rand() % 150;
+  }
+  return RET_OK;
+}
+
+void sensors_init(void)
```

Comment:
Need to call this in main somewhere still

Reply:
yep